ui/src/App.tsx

```diff
@@ -35,6 +35,9 @@ function App(): JSX.Element {
             <Route path="/upload">
               <Upload />
             </Route>
+            <Route path="/stories">
```

Comment:
Maybe have the path as "/all-stories" or "/allstories" (or something along those lines). This would be to avoid confusion with the /api/stories backend endpoint. If you think this isn't important though, it's fine as is.

Reply:
makes sense! 👍🏻